spec/cypress/app_commands/scenarios/basic.rb

```diff
@@ -0,0 +1,3 @@
+# You can setup your Rails state here
+# MyModel.create name: 'something'
+Post.create(title: 'I am a Postman')
```

Comment:
@skaidra-bake lets I think we can delete this as we are not yet ready to test this

Reply:
Oh yes, this does not exist anymore I don't think